src/main/java/rx/broadcast/InMemoryBroadcast.java

```diff
@@ -0,0 +1,27 @@
+package rx.broadcast;
+
+import rx.Observable;
+import rx.subjects.PublishSubject;
+import rx.subjects.Subject;
+
+public final class InMemoryBroadcast implements Broadcast {
+    private final Subject<Object, Object> values;
+
+    public InMemoryBroadcast() {
+        values = PublishSubject.create();
```

Comment:
Looking at it now, this should probably be wrapped in a [`SerializedSubject`](http://reactivex.io/RxJava/javadoc/rx/subjects/SerializedSubject.html)